json_schemas/briefs-digital-outcomes-and-specialists-digital-specialists.json

```diff
@@ -188,6 +194,7 @@
     "numberOfSuppliers",
     "organisation",
     "priceWeighting",
+    "requirementsLength",
```

Comment:
Yeah, I think setting all published ones to 2 weeks is what we want.

Running `python application.py db migrate --rev-id <number> -m <message>` should create a new migration file. We use numeric migration IDs and increment each number by 10 (so if the current version is 680 then new migration revision would be 690).

[Here's an example of a migration that changes a brief field](https://github.com/alphagov/digitalmarketplace-api/blob/master/migrations/versions/610_change_numberofsuppliers_to_integer.py).